kinto/__main__.py

```diff
@@ -4,7 +4,7 @@
 import sys
 
```

Comment:
I agree that we don't really need the "cliquet" script here and that the kinto script will be enough for us.